### ISSUE TITLE
Problems after 2022.3 HA update

### ISSUE THREAD

**Issue by luixal** [ldotlopez/ha-ideenergy#3]
Hi!

I just updated to Home Assistant 2022.3 and this integration is no longer working for me. This is what I can tell from the logs:

```
Logger: homeassistant.setup
Source: setup.py:298
First occurred: 7 de marzo de 2022 23:59:36 (1 occurrences)
Last logged: 7 de marzo de 2022 23:59:36

Unable to prepare setup for platform ideenergy.sensor: Platform not found (cannot import name 'MappingProxyType' from 'homeassistant.core' (/usr/src/homeassistant/homeassistant/core.py)).
```

Have tried to restart HA some times, but didn't seem to fix the issue.

Any ideas?

**Comment by ldotlopez** [comment]
Hi, can you try the "main" version from HACS? (or main branch from git).

I made some changes which work for me, if someone can confirm it's working I will release a 'named' version

**Comment by luixal** [comment]
Hi,

Just tried the _main_ version/branch and it's working fine :)

I guess you can just publish it as a new version, great work!

Thanks!

**Comment by ldotlopez** [comment]
Thanks for testing it!

I just released v0.2.1, you can go back from 'main' without problems.
